spec/providers/coreIntegration/tcpsocket.integration.src.js

```diff
@@ -115,21 +115,21 @@ module.exports = function (provider, setup) {
       var paused = false;
       var messageCount = 0;
       dispatch.on('onMessage', function (msg) {
-        if (!('data' in msg)) {
+        if (msg && !(msg.hasOwnProperty('data'))) {
           // Not an 'onData' message.
           return;
         }
 
         // One onData is allowed during pause due to https://crbug.com/360026
-        ++messageCount;
-        if (paused && messageCount === 1) {
+        if (paused && ++messageCount === 1) {
```

Comment:
Fair, not a strong preference but I figured I'd cut the line since we didn't need to use it as a message count later. Will rename to be more descriptive, or could switch back to how it was if you prefer.